web/src/ShareSnapshotModal.tsx

```diff
@@ -80,7 +80,7 @@ export default class ShareSnapshotModal extends PureComponent<props> {
       return (
         <section>
           <p>
-            Register on TiltCloud to share under your name, view, and delete
+            Connect Tilt to TiltCloud to share under your name and manage your
```

Comment:
Maybe: "Connect this Tilt install to TiltCloud"?

Reply:
I'd considered that but figured it felt excessively precise. Like, it's only relevant if you switch to another install, and at that point maybe you're not surprised if you need to again click connect to TiltCloud?